### ISSUE TITLE
[Feature request] Non-compressed video capture

### ISSUE THREAD

**Issue by Bucky21659** [MaartenBaert/ssr#180]
Hi, I'm a FRAPS user and I've been wanting to be able to switch to Linux but I haven't found a viable solution for recording games under my Ubuntu 14.04 install.  This seems very promising but there doesn't seem to be an option to disable compression (make it just write a lossless video file that I can compress using the video editor of choice).

If requesting this doesn't exactly make sense, allow me to elaborate.

When I record games on Windows with FRAPS, I open FRAPS, start the game, and start recording, there's not much performance impact since I have it set up to write to my secondary 2TB HDD which is mostly dedicated for recording (it's split 2 ways, 1TB is ext4, while the remaining space is NTFS).  I then will either re-encode and cut with VirtualDub (which works very well on WINE so I can use that on Linux as well), or if I want to add text/captions and/or transitions then I'll open it in a video editing program.

Basically, it'd be nice if people who have asinine amounts of free-space like me could have SSR use as much as it wants, then compress later to reduce the performance hit when recording.

Thanks!
